NEARFlags.json

```diff
@@ -0,0 +1,31 @@
+{
+  "EXAMPLE_FLAG": {
+    "createdBy": "Daryl Collins",
+    "createdAt": "2021-05-24T09:33:27.489Z",
+    "development": {
+      "enabled": true,
+      "lastEditedBy": "Daryl Collins",
```

Comment:
I personally feel that it is more convenient to have `git blame` as single source of truth for "when this was edited by who" (especially given how good GH UX around it is these days), but don't have hard stance if other people find it convenient cc @Patrick1904 @marcinbodnar @stefanopepe

Reply:
I agree git blame is awesome <3 :) This is mainly here so that we can easily build some basic tooling around this like 'how long as this flag been on in production env' and add something to prompt us to remove older flags that have not been toggled anywhere in a while :)